pkg/asset/installconfig/nutanix/validation.go

```diff
@@ -0,0 +1,63 @@
+package nutanix
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/pkg/errors"
+	"k8s.io/apimachinery/pkg/util/validation/field"
+
+	"github.com/openshift/installer/pkg/types"
+	nutanixtypes "github.com/openshift/installer/pkg/types/nutanix"
+	"github.com/openshift/installer/pkg/types/nutanix/validation"
+)
+
+// Validate executes platform-specific validation.
+func Validate(ic *types.InstallConfig) error {
+	if ic.Platform.Nutanix == nil {
+		return errors.New(field.Required(field.NewPath("platform", "nutanix"), "Nutanix validation requires a Nutanix platform configuration").Error())
```

Comment:
the platform is `nutanix` rather than `Nutanix`. Recommended change:
`
nutanix validation requires a nutanix platform configuration
`